eth_tester/main.py

```diff
@@ -463,7 +470,15 @@ def _add_transaction_to_pending_block(self, transaction):
             if is_locked:
                 raise AccountLocked("The account is currently locked")
 
-        raw_transaction_hash = self.backend.send_transaction(raw_transaction)
+        if {'r', 's', 'v'}.issubset(transaction.keys()):
+            if hasattr(self.backend, 'send_signed_transaction'):
```

Comment:
If you go ahead and add a stub implementation of the `send_signed_transaction` to the `eth_tester.backends.base.BaseChainBackend` you can instead use a `try/except` here, catching the `NotImplementedError` which feels a bit cleaner.